Tools/autotest/vehicle_test_suite.py

```diff
@@ -11404,10 +11404,8 @@ def SetpointGlobalPos(self, timeout=100):
             raise ValueError()
 
         def to_alt_frame(alt, mav_frame):
-            if mav_frame in ["MAV_FRAME_GLOBAL_RELATIVE_ALT",
-                             "MAV_FRAME_GLOBAL_RELATIVE_ALT_INT",
-                             "MAV_FRAME_GLOBAL_TERRAIN_ALT",
-                             "MAV_FRAME_GLOBAL_TERRAIN_ALT_INT"]:
+            # Per MAVLink spec, GLOBAL is a keyword to represent MSL.
+            if "GLOBAL" in mav_frame:
```

Comment:
This doesn't look right.

"GLOBAL" is refering to to the x/y fields being lat/lng, nothing to do with the alt, which is where the second part comes in; if not present it's MSL, if it is present it might eg. specify altitude is relative to origin or terrain.

Reply:
Thanks for the explanation. After reviewing that and the documenation, this makes sense. Closing this PR, behavior looks correct. as-is.